digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -312,6 +333,18 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     }
   }
 
+  delete("/massTransitStops/removal") {
+    val user = userProvider.getCurrentUser()
+    val assetId = (parsedBody \ "assetId").extractOpt[Int].get
+    massTransitStopService.getPersistedAssetsByIds(Set(assetId)).headOption.map{ a =>
+      a.linkId match {
+        case 0 => validateUserMunicipalityAccessByMunicipality(user)(a.municipalityCode)
+        case _ => validateUserMunicipalityAccessByLinkId(user, a.linkId, a.municipalityCode)
+      }
+      massTransitStopService.deleteMassTransitStopData(assetId)
+    }
+  }
+
```

Comment:
Entry point above is exactly the same.
Probably is to change to delete massServiceStops.